fastlane/Fastfile

```diff
@@ -48,7 +48,7 @@ platform :ios do
     run_tests(
       workspace: "AlphaWallet.xcworkspace",
       scheme: "AlphaWalletTests",
-      device: "iPhone X",
+      device: "iPhone Xʀ",
```

Comment:
Why does switching to a different device help?

Reply:
yep, it seems like i didn't understand task.